### PR TITLE
Add 1.9.5

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Comment by gbalabasquer** [comment]
It also includes adding missing links to 1.9.4 in the main page.
Take into account there is no 1.9.5 for kovan as the vesting contracts were deployed directly only to Goerli (which will have the changelog updated probably next week).
This PR still misses the ABIs file.

File: releases/mainnet/1.9.5/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.5</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.5/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmWXAVJVtp42HxPZrPGm6aTGYK4vXoW8Q1uuJVa31A82Md">Contract addresses (IPFS hosted)</a></li>
```
Comment (zdumitru) [inline releases/mainnet/1.9.5/index.html:13]: This doesn't open for me.

**Review by zdumitru — CHANGES_REQUESTED** [review]
And no 1.9.5 Kovan release this time?

**Review by brianmcmichael — APPROVED** [review]
Looks good. Checked new addresses against mainnet spell.

File: releases/mainnet/1.9.5/index.html (hunk 1 above)
Comment (brianmcmichael) [reply releases/mainnet/1.9.5/index.html:13]: Sometimes it takes a few seconds to find it on the network. It's working for me.

**Comment by gbalabasquer** [comment]
> And no 1.9.5 Kovan release this time?

Right, as I said in the prev message there won't be kovan release. Next week we will probably have the new Goerli in place with its 1.9.5 version.

File: releases/mainnet/1.9.5/index.html (hunk 1 above)
Comment (gbalabasquer) [reply releases/mainnet/1.9.5/index.html:13]: Yeah working here as well.

**Review by zdumitru — APPROVED** [review]

**Pull request merged**